runtime/lua/vim/_defaults.lua

```diff
@@ -22,7 +22,11 @@ do
   end, { desc = 'Inspect treesitter language tree for buffer', count = true })
 
   vim.api.nvim_create_user_command('EditQuery', function(cmd)
-    vim.treesitter.query.edit(cmd.fargs[1])
+    local opts = {}
+    if cmd.smods.horizontal then
+      opts.wincmd = 'new'
+    end
```

Comment:
Can this also support other split modifiers like `:botright horizontal EditQuery`, if not too difficult, by combining with `smods.split`?